plugins/content-highlight/package.json

```diff
@@ -48,7 +48,7 @@
     "typescript": "^5.4.5"
   },
   "peerDependencies": {
-    "blockly": "^11.0.0"
+    "blockly": "^11.1.2"
```

Comment:
This will need another bump before being merged correct?

Reply:
I'm not sure. The current release is v11.1.1, so I picked a higher number, but I don't know whether the next release will be a minor or patch release.